packages/mui-utils/src/getScrollbarSize/getScrollbarSize.ts

```diff
@@ -1,7 +1,10 @@
 // A change of the browser zoom change the scrollbar size.
 // Credit https://github.com/twbs/bootstrap/blob/488fd8afc535ca3a6ad4dc581f5e89217b6a36ac/js/src/util/scrollbar.js#L14-L18
-export default function getScrollbarSize(doc: Document): number {
+export default function getScrollbarSize(win?: Window | null): number {
   // https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth#usage_notes
-  const documentWidth = doc.documentElement.clientWidth;
-  return Math.abs(doc.defaultView!.innerWidth - documentWidth);
+  if (!win) {
+    win = window;
+  }
+  const documentWidth = win.document.documentElement.clientWidth;
+  return Math.abs(win.innerWidth - documentWidth);
```

Comment:
We could remove the `abs` here as per https://github.com/mui/material-ui/issues/43827#issuecomment-2364580040
```suggestion
  return win.innerWidth - documentWidth;
```